src/routes/timeline.nim

```diff
@@ -105,8 +105,21 @@ template respTimeline*(timeline: typed) =
     resp Http404, showError("User \"" & @"name" & "\" not found", cfg)
   resp t
 
+template respUserid*() = 
+  cond @"user_id".len > 0
+  let screen_name = await getCachedProfileScreenName(@"user_id")
+  if screen_name.len > 0:
+    redirect("/" & screen_name)
+  else:
+    resp Http404
+
 proc createTimelineRouter*(cfg: Config) =
   router timeline:
+    get "/i/user/@user_id":
+      respUserid()
+    get "/intent/user":
```

Comment:
Is this route missing a parameter? Also needs an empty line above it

Reply:
I'm not very familiar with Jester.

I think  "/i/user/@user_id"  and  "/intent/user"?user_id=xxx will both set @"user_id" parameter.

Reply:
Yeah I think that's correct, it's a little weird but Jester does indeed merge the URL parameters and query parameters.